browsermob-rest/src/main/java/net/lightbody/bmp/proxy/bricks/ProxyResource.java

```diff
@@ -478,28 +478,6 @@ public void process(BrowserMobHttpRequest request, Har har) {
             } catch (NumberFormatException e) {
             }
         }
-        String payloadPercentage = request.param("payloadPercentage");
```

Comment:
This functionality still works when using the legacy implementation, so we should keep this code in, even though the new implementation doesn't support it. Optionally, you could check to see if proxy is an instanceof BrowserMobProxyServer and log a warning that the feature is not supported with the new implementation.